tokio/src/task/mod.rs

```diff
@@ -307,11 +307,13 @@ cfg_rt! {
     mod unconstrained;
     pub use unconstrained::{unconstrained, Unconstrained};
 
+    pub mod join_set;
```

Comment:
This makes the `join_set` module public. Is that intentional? Is anything in it?

Reply:
That was a mistake, and it's been removed.

Reply:
Hmm, actually we do provide this as the only way to expose the builder. I will modify this to be public only with the unstable flag.

Reply:
Never mind I'll just expose the builder then.

Reply:
Done.